### DISCUSSION TITLE
re-appearing player after crossing the border (?)

### DISCUSSION THREAD

**Discussion by AgeTheGeek** [Quit975/DuckyGame#40] [Ideas]
New mechanic idea:

- when player leaves through the right border, while still moving right, he appears again from the left border (and like-wise)
- when player leaves through the bottom border, while still moving down, he appears again from the up border (and like-wise)

conclusion: infinite movement range closed within the size of game's window

**Comment by Quit975** [comment]
I even thought of the very same thing recently and thought that it would be very tricky thing to do xD and instead thought that we will probably need some graphical representation of walls to clearly indicate where are the borders :D

I will turn it into a discussion for now - once we have the prototype ready and we will start iterating over game mechanics, we will go through all the ideas we have there and if we will want to implement it, we will turn it into the issue
